src/components/page-donate/page-donate.tsx

```diff
@@ -239,22 +239,19 @@ export class PageDonate {
                   <button
                     onClick={handleCheckout}
                     class={"button is-red is-fullwidth-mobile " + (!this.amount || isNaN(this.amount) ? "is-disabled" : "")}
-                    disabled={!this.amount || isNaN(this.amount)}
+                    disabled={!this.amount || isNaN(this.amount) || (this.error || "").length > 0}
                   >
                     Donate
                   </button>
                   {this.error && (
                     <div id="donation-error" class="help has-text-red">
                       <p>{this.error}</p>
+                      <a class="button is-blue is-fullwidth-mobile" target="_blank" href={`https://paypal.me/pizzatothepolls/${this.amount}`}>
```

Comment:
Since it errored, maybe check for it not being set just in case?
`https://paypal.me/pizzatothepolls/${this.amount || ""}`